src/schema/v2/homeView/sections/GalleriesNearYou.ts

```diff
@@ -19,4 +20,14 @@ export const GalleriesNearYou: HomeViewSection = {
   },
   ownerType: OwnerType.galleriesForYou,
   requiresAuthentication: false,
+
+  resolver: withHomeViewTimeout(async (_parent, _args, _context, _info) => {
+    return {
+      title: "Galleries Near You",
+      subtitle: "Follow these local galleries for updates on artists you love.",
+      href: "/galleries-for-you",
+      buttonText: "Explore",
+      image_url: "https://files.artsy.net/images/galleries_for_you.webp",
+    }
+  }),
```

Comment:
**suggestion**: I'm wondering if we can reuse some of the parent attributes for the card details here:

```suggestion
  resolver: withHomeViewTimeout(async (parent, _args, _context, _info) => {
    return {
      title: parent.title,
      subtitle: parent.description,
      href: "/galleries-for-you",
      buttonText: "Explore",
      image_url: "https://files.artsy.net/images/galleries_for_you.webp",
    }
  }),
```

Reply:
We might also consider removing `component>backgroundImageURL` and `component>behaviors` in this section instance if we're expecting the client to use the card data defined here instead.

Reply:
Nice idea, added in https://github.com/artsy/metaphysics/pull/6116/commits/8795f53c16e0680c0bc275af619960273d3e3f25